integration/setup/common.go

```diff
@@ -93,11 +98,65 @@ func SkipForPostgresWithReason(tb testing.TB, reason string) {
 	}
 }
 
+// buildMongoDBURIOpts represents buildMongoDBURI's options.
+type buildMongoDBURIOpts struct {
+	hostPort       string
+	unixSocketPath string
+	tls            bool
+}
+
+// buildMongoDBURI builds MongoDB URI with given URI options.
+func buildMongoDBURI(tb testing.TB, opts *buildMongoDBURIOpts) string {
+	var host, path string
+
+	if opts.hostPort != "" {
+		require.Empty(tb, opts.unixSocketPath, "both hostPort and unixSocketPath are set")
+		host = opts.hostPort
+		path = "/"
+	} else {
+		require.NotEmpty(tb, opts.unixSocketPath, "neither hostPort nor unixSocketPath are set")
+		path = opts.unixSocketPath
+	}
+
+	q := make(url.Values)
+
+	if opts.tls {
```

Comment:
How would tls work in case of unix socket configuration? Based on the current implementation, it looks like it's a possible case. Is it?

Reply:
I've added the check for the Unix socket.